src/components/GoalsComponents/GoalConfigModal/ConfigGoal.tsx

```diff
@@ -149,7 +150,23 @@ const ConfigGoal = ({ goal, action }: { action: "Update" | "Create"; goal: GoalI
       }
       suggestChanges(rootGoal, getFinalTags(), title, goalColor, subGoalsHistory.length);
     } else {
-      await modifyGoal(goal.id, getFinalTags(), title, goalColor, [...ancestors, goal.id], hints);
+      updateStatus = await modifyGoal(goal.id, getFinalTags(), title, goalColor, [...ancestors, goal.id], hints);
+    }
+
+    if (updateStatus) {
```

Comment:
you can use ternary operator here as we need to show toast anyway

Reply:
Noted tushar I will fix this